Gemfile

```diff
@@ -4,6 +4,7 @@ source 'https://rubygems.org'
 
 gem 'rails', '~> 6.0.0'
 gem 'rails-i18n', '~> 6.0.0'
+gem 'webpacker', '~> 4.0'
```

Comment:
the gem should be also in https://github.com/thredded/thredded/blob/master/spec/gemfiles/rails_6_0.gemfile ?

ideally you should test both variants
1/ rails 6 + webpack
2/ rails 6 + sprockets 4

Reply:
Done